quesma/model/README.md

```diff
@@ -15,7 +15,7 @@ More info: https://www.elastic.co/guide/en/elasticsearch/reference/current/searc
  Cardinality               |   :white_check_mark:   | Auto-interval date histogram |    :wavy_dash:     | Bucket script          |     :wavy_dash:    |
  Extended Stats            | :white_check_mark:[^1] | Categorize text              |        :x:         | Bucket count K-S test  |        :x:         |
  Avg                       |   :white_check_mark:   | Children                     |        :x:         | Bucket correlation     |        :x:         |
- Boxplot                   |          :x:           | Composite                    |        :x:         | Bucket selector        |        :x:         |
+ Boxplot                   |          :x:           | Composite                    | :white_check_mark: | Bucket selector        |        :x:         |
```

Comment:
Unrelated to this PR, but maybe we should move this table to our public-facing docs?

Reply:
I think so too!

Reply:
It does not look like composite works now with arrays as they described. Though still major step forward.